src/pagination/pagination.styles.tsx

```diff
@@ -101,98 +94,49 @@ export const NavigationButton = styled(ClickableIcon)<ButtonProps>`
     background: ${(props) =>
-        props.$selected ? V2_Color.Primary : V2_Color.Neutral[8]};
-    border: 1px solid
-        ${(props) => (props.$selected ? V2_Color.Primary : V2_Color.Neutral[5])};
+        props.$selected ? Colour["bg-primary"] : Colour.bg};
+    border: 1px solid ${(props) => (props.$selected ? "" : Colour.border)};
     color: ${(props) =>
-        props.$selected ? V2_Color.Neutral[8] : V2_Color.Neutral[1]};
+        props.$selected ? Colour["text-inverse"] : Colour.text};
 
     min-width: 2.5rem;
-    height: 2.5rem;
     text-align: center;
     padding: 0.4rem 0.5rem;
     border-radius: 0.25rem;
     margin: 0.25rem;
     cursor: pointer;
 
-    span {
-        color: ${(props) =>
-            props.$selected ? V2_Color.Neutral[8] : V2_Color.Neutral[1]};
-        ${(props) => {
-            if (props.$selected) {
-                return css`
-                    ${V2_TextStyleHelper.getTextStyle("Body", 700)};
-                `;
-            } else {
-                return css`
-                    ${V2_TextStyleHelper.getTextStyle("Body", 400)};
-                `;
-            }
-        }}
-    }
+    ${(props) =>
+        props.$selected
+            ? css`
+                  ${Font["body-baseline-bold"]}
+              `
+            : css`
+                  ${Font["body-baseline-regular"]}
+              `}
+
     :hover {
         box-shadow: none;
-
-        background: ${(props) =>
-            props.$selected ? V2_Color.Primary : V2_Color.Accent.Light[5]};
-        border: 1px solid
-            ${(props) =>
-                props.$selected ? V2_Color.Primary : V2_Color.Accent.Light[5]};
-        span {
-            color: ${(props) =>
-                props.$selected ? V2_Color.Neutral[8] : V2_Color.Primary};
-            ${(props) => {
-                if (props.$selected) {
-                    return css`
-                        ${V2_TextStyleHelper.getTextStyle("Body", 700)};
-                    `;
-                } else {
-                    return css`
-                        ${V2_TextStyleHelper.getTextStyle("Body", 600)};
-                    `;
-                }
-            }}
-        }
-    }
-
-    :active,
-    :focus {
-        background: ${(props) =>
-            props.$selected ? V2_Color.Primary : V2_Color.Neutral[8]};
-        outline: none;
-        box-shadow: inset 0px 0px 4px 1px ${V2_Color.Accent.Light[1]};
-        span {
-            color: ${(props) =>
-                props.$selected ? V2_Color.Neutral[8] : V2_Color.Primary};
-
-            ${(props) => {
-                if (props.$selected) {
-                    return css`
-                        ${V2_TextStyleHelper.getTextStyle("Body", 700)};
-                    `;
-                } else {
-                    return css`
-                        ${V2_TextStyleHelper.getTextStyle("Body", 400)};
-                    `;
-                }
-            }}
-        }
+        border: none;
```

Comment:
instead of none, set the same colour as the background to ensure the button height remains the same

```suggestion
        border: ${Colour["bg-hover"]};
```